docs/2.0. EDID to Receiver Caps mapping.md

```diff
@@ -1,84 +1,192 @@
+
+Similarly, [BCP-004-01][BCP-004-01] Receiver Capabilities defines a methodology for describing the Receivers with constraints on the properties of streams which related to [IS-04][IS-04] Senders.
 
 ## Scope
 
-This document provides Best Current Practice guidelines for how to implement a mapping of EDID fields to Receiver Capabilities in order to have a specific way of representing an EDID sink devices through an associated IS-04 Receiver.
+This document provides Best Current Practice guidelines for how to implement a mapping of EDID fields to Receiver Capabilities in order to have a specific way of representing an EDID sink devices through an associated [IS-04][IS-04] Receiver.
 
 ## Use of Normative Language
 
 The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119][RFC-2119].
 
+## Normative References
+
+These appear at the end of the Markdown source for this document, and are referenced as hyperlinks within the main body.
+
 ## Definitions
 
 _See also the [NMOS Glossary](https://specs.amwa.tv/nmos/branches/main/docs/4.0._Glossary.html), and definitions within RFCs._
 
 ### Gateway Device
 
-A device which encapsulates a digital video or/and audio signal (usually, from a physical connection) into IP packets. Conversely, the device may perform the reverse operation for IP packets to a digital signal.
+A device which encapsulates a digital video or/and audio signal (usually, from a physical connection) into IP packets. Conversely, the device MAY perform the reverse operation for IP packets to a digital signal.
 
 ### Sink
 
-A media consuming unit associated with an IS-04 Receiver.
+A media consuming unit described by an EDID which can be associated to a Receiver.
 
 ## Introduction
 
-The mapping of a Sink's EDID to a Receiver's Capabilities is limited to Devices that treat signals as immutable. Devices capable of altering a signal MUST reflect the capabilities of the Receiver including any types of conversion or restrictions. Devices which support multiple Sinks per Receiver MUST expose the capabilities of the Receiver if any internal processing occures (e.g. the Receiver is able to consume 4K Flow and downscale it for the Sink who's EDID describes support up to HD resolutions).
+Gateway Devices which bridge a NMOS Receivers to baseband connection need to expose the capabilities of the baseband interface through the NMOS resouce to enable connection management. This can be obtained by mapping a Sink's EDID to a Receiver's Capabilities and factoring the abilities of the Gateway Device.
```

Comment:
```suggestion
Gateway Devices which bridge a NMOS Receivers to baseband connection need to expose the capabilities of the baseband interface through the NMOS resource to enable connection management. This can be obtained by mapping a Sink's EDID to a Receiver's Capabilities and factoring the abilities of the Gateway Device.
```